platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDbDataSource.java

```diff
@@ -492,35 +414,35 @@ public long getLastLeafPath() {
     }
 
     /**
-     * Pauses merging of all data file collections used by this data source. It may not stop merging
-     * immediately, but as soon as merging process needs to update data source state, which is
+     * Pauses compaction of all data file collections used by this data source. It may not stop compaction
+     * immediately, but as soon as compaction process needs to update data source state, which is
      * critical for snapshots (e.g. update an index), it will be stopped until {@link
-     * #resumeMerging()}} is called.
+     * #resumeCompaction()}} is called.
      */
-    void pauseMerging() throws IOException {
+    void pauseCompaction() throws IOException {
         if (hasDiskStoreForHashes) {
-            hashStoreDisk.pauseMerging();
+            hashStoreDisk.pauseCompaction();
         }
-        pathToKeyValue.pauseMerging();
+        pathToKeyValue.pauseCompaction();
         if (!isLongKeyMode) {
             objectKeyToPath.pauseMerging();
         }
     }
 
     /** Resumes previously stopped data file collection merging. */
-    void resumeMerging() throws IOException {
+    void resumeCompaction() throws IOException {
         if (hasDiskStoreForHashes) {
-            hashStoreDisk.resumeMerging();
+            hashStoreDisk.resumeCompaction();
         }
-        pathToKeyValue.resumeMerging();
+        pathToKeyValue.resumeCompaction();
         if (!isLongKeyMode) {
             objectKeyToPath.resumeMerging();
         }
     }
 
     /**
      * Save a batch of data to data store.
-     *
+     * <br>
```

Comment:
Is it accepted by spotless? As far as I can remember, it usually requires `<p>` in the beginning of each new paragraph rather than `<b>`

Reply:
replaced with <p>